build-aux/main.mk

```diff
@@ -171,3 +184,14 @@ endif
 	$(foreach src,$(filter %.yaml,$^),$(foreach dst,$(patsubst docs/yaml/%,$@/%,$(src)),\
 	  mkdir -p $(dir $(dst))$(NL)\
 	  sed -e 's/\$$version\$$/$*/g' -e 's/\$$quoteVersion$$/0.4.1/g' <$(src) >$(dst)$(NL)))
+
+#
+# Destructive rules
+
+clobber: clean
+
+clean: build-output.rm-r
```

Comment:
You know, we have defaults for this in common.mk... I was considering importing that in the `Makefile`.

Reply:
builder.mk will get upset with common.mk